Dockerfile

```diff
@@ -3,12 +3,10 @@ ADD . src/github.com/6f7262/kipp
 RUN apk update && \
     apk add git build-base && \
     go get -v github.com/6f7262/kipp/cmd/kipp && \
-    go build -v -o /kipp github.com/6f7262/kipp/cmd/kipp
+    go build -v -ldflags "-linkmode external -extldflags -static" -o 
```

Comment:
Did you try this? Does it actually work?

Reply:
Inside the Dockerfile, that is. Docker build works, docker run works?

Reply:
Yes, it built and ran correctly.

Reply:
Doesn't build for me. ![](https://up.sq10.net/c734683d0d561319dd9e2.png)

Reply:
I see, it's an issue with then I pasted it into nano. There a new line after -o. Do you mind changing it? I'm not home right now.

Reply:
nano's word wrapping A++

committed fix, gonna test this locally